src/styles/ui/CardButton.scss

```diff
@@ -26,6 +26,10 @@
   // text-align: center;
   align-items: flex-start;
   display: inline-flex;
+  font-weight: 700;
```

Comment:
Do you want to do that to ALL cards or just one type of cards? If the later, maybe use id or classname